docs/Using-Fleet/Adding-hosts.md

```diff
@@ -1,32 +1,34 @@
 # Adding hosts
 
-- [Introduction](#introduction)
-- [Add hosts with Orbit](#add-hosts-with-orbit)
-  - [Signing installers](#signing-installers)
-  - [Including Fleet Desktop](#including-fleet-desktop)
-  - [Adding multiple hosts](#adding-multiple-hosts)
-  - [Automatically adding hosts to a team](#automatically-adding-hosts-to-a-team)
-  - [Configuration options](#configuration-options)
-- [Add hosts with plain osquery](#add-hosts-with-plain-osquery)
-  - [Set an environment variable with an enroll secret](#set-an-environment-variable-with-an-enroll-secret)
-  - [Deploy the TLS certificate that osquery will use to communicate with Fleet](#deploy-the-tls-certificate-that-osquery-will-use-to-communicate-with-fleet)
-  - [Launching osqueryd](#launching-osqueryd)
-  - [Using a flag file to manage flags](#using-a-flag-file-to-manage-flags)
-    - [Flag file on Windows](#flag-file-on-windows)
-  - [Migrating from plain osquery to osquery installer](#migrating-from-plain-osquery-to-osquery-installer)
-    - [Generate installer](#generate-installer)
-    - [Migrate](#migrate)
-- [Grant full disk access to osquery on macOS](#grant-full-disk-access-to-osquery-on-macos)
-  - [Creating the configuration profile](#creating-the-configuration-profile)
-    - [Obtaining identifiers](#obtaining-identifiers)
-    - [Creating the profile](#creating-the-profile)
-    - [Test the profile](#test-the-profile)
+- [Adding hosts](#adding-hosts)
```

Comment:
@lucasmrod, would it be possible to delete the "Adding hosts" bullet and then move the rest of the bullets up a tier without breaking these links? Including a link for Adding hosts seems unnecessary since this list appears under the Adding hosts heading.